setup/README.md

```diff
@@ -0,0 +1,33 @@
+# Mondoo Setup Action
```

Comment:
I feel like this is a little weird type of an action. If we want to have an action that does the installation of the Mondoo client and that's all it does, then this is not it. If I want to only install the client I would expect that none of the properties specified in the table below are actually needed. All I am doing is installing the client and not using it.

I can see perhaps the use of `service-account-credentials` if I also want to configure it to use my Mondoo space during install but that would be it. I don't see any use case at all of running any command with the `args` property, for example. If I want to do get the status I would rather do:
```yaml
- run: mondoo status
```
instead of trying to do the same via this action's interface

Reply:
@imilchev The setup action provides an easy way to install Mondoo Client into any existing workflow, and then the ability to send any supported commands to the client. The overall design is to have very prescriptive actions for specific tasks, and this setup action for any custom use case. 

@chris-rock is working to support the ability to configure the service account/config via an environment variable, rather than a path to the config. 

I see your point about the `mondoo status` step. I can remove that.

Reply:
I see the need for a Mondoo installer action. I still doubt the benefits of allowing to implicitly also run a command with the installer action. For example, if I have a "weird" use-case where I want to run >1 Mondoo command in my workflow the documentation implies I would need to go through the installation script twice. That is not necessary.

Have a look at the HashiCorp Terraform setup action [here](https://github.com/hashicorp/setup-terraform). The action does installation and configuration but actually Terraform commands are run via a simple `run` step

Reply:
I think for this action specifically it is interesting to allow pinning a Mondoo Client version. Here a user is "manually" executing Mondoo commands, which means that silently installing a newer version of the client can very easily break the entire workflow.

Reply:
@imilchev I feel like the version pinning is great to call out, but also a documentation issue more than anything.